content/profiles/jonah-andersson.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Jonah Andersson"
+profileimage: "https://azdurablefunctionstorage.blob.core.windows.net/blogcontent/JonahAndersson_ProfilePhoto.jpg"
+coverimage: "https://azcdnendpointjonahanderssontech.azureedge.net/wp-content/uploads/2020/05/CoverPageStartPage.png"
+tags: ["CSharp", "Public Speaking","Azure Developer","Programmer","Mentorship", "AzureMVP", "DotNet", "Fullstack", "Microsoft Certified Trainer"]
+twitter: "http://www.twitter.com/cjkodare"
+github: "https://github.com/jonahandersson"
+twitch: ""
+instagram: "https://instagram.com/jonahandersson_ladyprogrammer"
+blogsite: "https://jonahandersson.tech/"
+linkedin: "https://www.linkedin.com/in/jonahandersson"
+comments: false
+summary: "I am a Microsoft MVP for Azure, Software Engineer, Certified Azure Developer, Microsoft Certified Trainer
+Tech Mentor, Public Speaker, Author, Blogger. Genuine, Humble, Brave, Sincere. Love Learning! 
+"
+draft: true
```

Comment:
Hey Jonah!
This line needs to be deleted for the profile to go live :-)

Reply:
I did it for you ;-)

Reply:
Thank you @Layla-P for updating and editing it.